app/views/pages/report.ejs

```diff
@@ -555,4 +737,276 @@
   var obfuscatePath = true;
 </script>
 
+<script>
+  /* Page validation functions. 
+  If validation fails and messages to be displayed should be dispatched in the validation function.
+  These aren't done */
+  const validateDemographicsAndGeography = () => {
```

Comment:
In order to support most browsers, we either need to:
- Write code that is supported by most broswers (as far as I can see, we can get away with changing `const`/`let` to `var`, and using normal functions instead of arrow functions).
- Move the script to a separate file, and add a `build:js` script to package.json as well that runs webpack or something similar for transpiling the code.

Reply:
replaced all const/let, arrow functions, and template literals. Of course this doesn't include those syntax uses within ejs parts of the view.